src/app/first-page/first-page.component.html

```diff
@@ -0,0 +1 @@
+<h1>First-page works!</h1>
```

Comment:
Not sure about the naming here. Can we name this "landing page" or "home"?

Reply:
I've renamed it as "landing-page"